### DISCUSSION TITLE
PICO W - read wifi signal (in client mode)

### DISCUSSION THREAD

**Discussion by JohnMTu** [micropython/micropython#9504] [General]
Hello,

is there a way to read wifi signal on rpi pico W ?

Way i know from ESPs was 
```sta_if.status('rssi')```

but does't work here.
thanks a lot
best regards
jan

**Comment by jimmo** [comment]
This is not currently supported on boards using the cyw43 wifi chip (Pico W, PYBD, Portenta). Might be something we can add in the future though!
